src/browser/actions/requestsActions.js

```diff
@@ -1,7 +1,8 @@
 export const ENQUEUE_REQUEST = 'ENQUEUE_REQUEST';
 export const DEQUEUE_REQUEST = 'DEQUEUE_REQUEST';
 export const EMPTY_REQUESTS = 'EMPTY_REQUESTS';
-
+export const EMPTY_ALL_REQUESTS = 'EMPTY_ALL_REQUESTS';
```

Comment:
:neckbeard: Can you leave a line of whitespace between the string constants and the function constants?  I think it will improve "scannability" for the file. 😄